p2p/simulations/network.go

```diff
@@ -595,8 +585,21 @@ type Node struct {
 	// Config if the config used to created the node
 	Config *adapters.NodeConfig `json:"config"`
 
-	// Up tracks whether or not the node is running
-	Up bool `json:"up"`
+	// up tracks whether or not the node is running
+	up   bool         `json:"up"`
```

Comment:
Unexported filed cannot be json-encoded, and encoding is handled in UnmarshalJSON. Json tags can be removed from this struct.

Reply:
:+1: fixed